cmd/resources/resources.go

```diff
@@ -0,0 +1,145 @@
+package resources
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+
+	"ldcli/cmd/cliflags"
+	"ldcli/cmd/validators"
+)
+
+func NewResourceCmd(parentCmd *cobra.Command, resourceName, shortDescription, longDescription string) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   resourceName,
+		Short: shortDescription,
+		Long:  longDescription,
+		//TODO: add tracking here
```

Comment:
actually not sure if we wanna add tracking when just the resource cmd is run? maybe we do?

Reply:
I've been adding them as `PersistentPreRun`. That means they won't run when just the resource command runs. But they'll run when subcommands on the resource run. This is what that [looks like on my branch/PR](https://github.com/launchdarkly/ldcli/blob/f3681bb62bc51e0a78c386a6a8ddb52967343f14/cmd/flags/flags.go#L15-L17)

Reply:
yup there's a comment in there - will make sure to add it in there!